### ISSUE TITLE
Capitalized words (i.e. The) from Ignored list causes validation to fail

### ISSUE THREAD

**Issue by amimas** [textlint-rule/textlint-rule-unexpanded-acronym#8]
The following line fails the validation by this linter:

```markdown
| `verification_value` | Required | `string` | The Card Verification Value (CVV) code from the back of the card |
```

Even though the acronym is expanded, it still is reported as an unexpanded acronym because the rule is comparing `CVV` to `TCVV`

If I change the above snippet of content to either one of these lines, the content passes the validation:

```markdown
| `verification_value` | Required | `string` | Card Verification Value (CVV) code from the back of the card |
| `verification_value` | Required | `string` | This is Card Verification Value (CVV) code from the back of the card |
```

Not sure if this issue is related to PR #6

**Comment by azu** [comment]
`A`, `An`, `The` are special words.
Anything else?

Probably, we need to ignore `A`, `An`, and `The` in this step.
❓ We also need to check the word is at first in the sentence?
https://github.com/textlint-rule/textlint-rule-unexpanded-acronym/blob/530d17b8088cc1fdc58683cfdfd075f6f4a5ee17/src/textlint-rule-unexpanded-acronym.js#L55-L59

As a result,

- `The Card Verification Value ` -> `CVV`
- `An American Company` -> `AC`
- `The A Word Word` -> `AWW` ? (I dont have that pattern)

What do you think about it?

**Comment by amimas** [comment]
I just tried the latest version that was released in PR #6 and that fixed the issue I posted in the original message.

I think we can close this as fixed. What do you think?

**Comment by azu** [comment]
ok
